_sass/_base.scss

```diff
@@ -108,16 +150,16 @@ a {
  * Blockquotes
  */
 blockquote {
-    color: $grey-color;
-    border-left: 4px solid $grey-color-light;
-    padding-left: $spacing-unit / 2;
-    font-size: 18px;
-    letter-spacing: -1px;
-    font-style: italic;
-
-    > :last-child {
-        margin-bottom: 0;
-    }
+  color: $grey-color;
```

Comment:
Properties should be ordered border-left, color, font-size, font-style, letter-spacing, padding-left